proto/couler.proto

```diff
@@ -66,21 +72,29 @@ message Step {
   ContainerSpec container_spec = 4;
   ResourceSpec resource_spec = 5;
   string script = 6;
-  repeated StepIO inputs = 7;
-  repeated StepIO outputs = 8;
-  repeated int32  dependencies = 9;
+  repeated StepIO args = 7;
+  repeated int32  dependencies = 8;
 }
 
 message ConcurrentSteps {
   repeated Step steps = 1;
 }
 
+message StepTemplate {
+  string name = 1;
+  repeated StepIO inputs = 2;
+  repeated StepIO outputs = 3;
+  // TODO(typhoonzero):
+  // add timeout, retry, pool, enable_ulogfs, daemon
```

Comment:
Remove pool and enable_ulogfs since these are internal keywords and are not used here.